### PR TITLE
Move all the autoyast profiles from Migration job groups to data/yam/

### PR DESCRIPTION
**Pull request by hjluo**
- Description
Move all the autoyast profiles from Migration job groups to data/yam/

- Related ticket: 
   - [**131336**](https://progress.opensuse.org/issues/131336)
- Related MR: 
    - [**310**](https://gitlab.suse.de/coolgw/wegao-test/-/merge_requests/310)
    - [**561**](https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/561)
- Needles:  N/A
- Verification run: 
   - [**All_Verification_Runs**](https://openqa.suse.de/tests/overview?arch=&flavor=&machine=&test=&modules=&module_re=&distri=sle&version=&build=hjluo%2Fos-autoinst-distri-opensuse%23move_autoyast_profile#)

### CODE REVIEW

**Comment by lemon-suse** [comment]
We should move these autoyast profile in MU-dev job group for support image to data/yam also.

**Comment by lemon-suse** [comment]
It seems you missed the autoyast_sle15/autoyast_scc_up_ppc64le.xml  in https://openqa.nue.suse.com/tests/11164722#settings

And please double check migration job groups for autoyast profile used. Thanks.

**Comment by hjluo** [comment]
> We should move these autoyast profile in MU-dev job group for migration support image to data/yam also.
> 
> Thought a while, these should be put in your another PR for support images.

OK, thank you.

File: data/yam/create_hdd_gnome_allpatterns_aarch64.xml
Comment (lemon-suse) [inline data/yam/create_hdd_gnome_allpatterns_aarch64.xml:1]: For these installation profile prepared for migration, we'd better to make them in another PR for support_images and move to data/yast/support_image/.

**Comment by lemon-suse** [comment]
Refer to PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17352, we should move these profiles to data/yam/autoyast/.

**Comment by hjluo** [comment]
> Refer to PR #17352, we should move these profiles to data/yam/autoyast/.

OK, thank you

**Comment by jknphy** [comment]
> > Refer to PR #17352, we should move these profiles to data/yam/autoyast/.
> 
> OK, thank you, @jknphy, just want to make sure that for migration, we need to move all migration-related autoyast profiles to data/yam/autoyast ? thank you

yes, that is the folder.
regarding the files that are in this PR, looks like they are for creation of images, or those are used for migration?
there should be a easy way to distinguish migration from installation easly by the name, please think about that.

File: data/yam/create_hdd_gnome_allpatterns_aarch64.xml
Comment (hjluo) [reply data/yam/create_hdd_gnome_allpatterns_aarch64.xml:1]: changed and please take a look again. Thank you

**Comment by hjluo** [comment]
> > > Refer to PR #17352, we should move these profiles to data/yam/autoyast/.
> > 
> > 
> > OK, thank you, @jknphy, just want to make sure that for migration, we need to move all migration-related autoyast profiles to data/yam/autoyast ? thank you
> 
> yes, that is the folder. regarding the files that are in this PR, looks like they are for creation of images, or those are used for migration? there should be a easy way to distinguish migration from installation easly by the name, please think about that.

changed and thank you.

**Comment by lemon-suse** [comment]
It seems you move all the profile to data/yam/autoyast/, but we should distinguish them from profile for migration or profile for installation for support images, for migration we should move to data/yam/autoyast/, for support images then move to data/yam/autoyast/support_images/.

**Comment by hjluo** [comment]
> It seems you move all the profile to data/yam/autoyast/, but we should distinguish them from profile for migration or profile for installation for support images, for migration we should move to data/yam/autoyast/, for support images then move to data/yam/autoyast/support_images/.

OK, I'd change them based on their names and rerun all the VRs to make sure they still work.

File: data/yam/support_images/create_hdd_gnome_allpatterns_aarch64.xml
Comment (lemon-suse) [inline data/yam/support_images/create_hdd_gnome_allpatterns_aarch64.xml:1]: Note, the folder of autoyast_sle15/create_hdd owned by functional squad, we should copy profiles from this folder to yam/autoyast/support_images while not move. Same for below other profiles.

Comment (hjluo) [reply data/yam/support_images/create_hdd_gnome_allpatterns_aarch64.xml:1]: @lemon-suse  @jknphy  so can you please help me to revert it?
